### PR TITLE
Fix section number comments thrown off by ExponentiationExpression addition; pluralize Keyed Collections to match Indexed Collections.

### PR DESCRIPTION
**Pull request by BrendanEich**
Fix section number comments thrown off by ExponentiationExpression addition; pluralize Keyed Collections to match Indexed Collections.

### CODE REVIEW

File: spec.html [hunk 1]
```diff
@@ -32328,8 +32332,8 @@
 </emu-clause>
 
 <!-- es6num="23" -->
-<emu-clause id="sec-keyed-collection">
-  <h1>Keyed Collection</h1>
+<emu-clause id="sec-keyed-collections">
```
Comment (annevk) [inline spec.html]: Should prolly not change the ID as that breaks links.

File: spec.html [hunk 2]
```diff
@@ -12605,6 +12605,7 @@
     </emu-clause>
   </emu-clause>
 
+  <!-- es6num="12.6" -->
```
Comment (annevk) [inline spec.html]: es6num is just a reference to the legacy section number in ES6. They should not be added or modified.

**Comment by BrendanEich** [comment]
Never mind, I'll just fix the h1 text to use the plural. Thanks again @annevk -- you got me on the right path for material changes to come.

**Comment by bterlson** [comment]
I'm fine leaving behind an anchor with the old id, but in this case seems fine to leave it as is.

**Pull request closed without merging**